src/v/cluster/partition_manager.cc

```diff
@@ -35,6 +35,17 @@ partition_manager::partition_manager(
   , _raft_manager(raft)
   , _tx_gateway_frontend(tx_gateway_frontend) {}
 
+partition_manager::ntp_table_container
+partition_manager::copy_table(const model::topic_namespace& tn) {
```

Comment:
Maybe this name can be more specific like `get_topic_partition_table` or similar ? Also maybe it would be better to use `node_hash_map` ?

Reply:
I just use the underlying type `ntp_table_container` you think it should be modified to `node_hash_map`?

Reply:
I am thinking about it causing memory fragmentation.